ZFHIFI/iocBoot/iocZFHIFI-IOC-01/config.xml

```diff
@@ -3,18 +3,16 @@
  <config_part>
   <ioc_desc>ISIS Muon Zero Field Controller</ioc_desc>
     <macros>
-    <macro name="MAGNETOMETER" pattern="^[0-9a-zA-Z:\._]+$" description="Magnetometer PV prefix, e.g. IN:INSTNAME:ZFMAGFLD_01" hasDefault="NO" />
-
-    <macro name="PSU_X" pattern="^[0-9a-zA-Z:\._]+$" description="X axis power supply prefix, e.g. IN:INSTNAME:CRYOSMS_01" hasDefault="NO" />
-    <macro name="PSU_Y" pattern="^[0-9a-zA-Z:\._]+$" description="Y axis power supply prefix, e.g. IN:INSTNAME:CRYOSMS_01" hasDefault="NO" />
-    <macro name="PSU_Z" pattern="^[0-9a-zA-Z:\._]+$" description="Z axis power supply prefix, e.g. IN:INSTNAME:CRYOSMS_01" hasDefault="NO" />
+    <macro name="PSU_X" pattern="^.*$" description="X axis power supply prefix, e.g. IN:INSTNAME:CRYOSMS_01" hasDefault="NO" />
+    <macro name="PSU_Y" pattern="^.*$" description="Y axis power supply prefix, e.g. IN:INSTNAME:CRYOSMS_01" hasDefault="NO" />
+    <macro name="PSU_Z" pattern="^.*$" description="Z axis power supply prefix, e.g. IN:INSTNAME:CRYOSMS_01" hasDefault="NO" />
 	
```

Comment:
what was the problem with the original pattern?

Reply:
HIFI-CRYOMAG has pv prefix `IN:HIFI-C11:` (i.e. contains a dash). If you prefer we could just add a dash as allowed.